package.json

```diff
@@ -52,7 +52,8 @@
   },
   "resolutions": {
     "@typescript-eslint/eslint-plugin": "^4.18.0",
-    "@babel/types": "7.25.6"
+    "@babel/types": "7.25.6",
+    "@joshwooding/vite-plugin-react-docgen-typescript": "0.3.0"
```

Comment:
This is to work around a storybook bug that slows down fast refresh. https://github.com/storybookjs/storybook/issues/28269